modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -391,6 +391,22 @@ public static boolean isNestedLoopRunning() {
         return Toolkit.getToolkit().isNestedLoopRunning();
     }
 
+    /**
+     * Indicates whether a nested event loop can be started from the current thread in the current state.
+     * Note that a nested event loop is not allowed outside an event handler.
```

Comment:
I would either remove this sentence (probably cleanest), or else change it to a positive statement (say when it is allowed, not when it isn't) and also add that it's ok when called from a Runnable passed to `Platform.runLater`. Maybe something like this:

```
A nested event loop can be started from an event handler or from a `Runnable` passed
to `Platform.runLater(Runnable)`.
```

Reply:
done.